bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/target/env/GuideGroup.java

```diff
@@ -155,7 +155,7 @@ public GuideGroup setName(String name) {
      * instance; <code>false</code> otherwise
      */
     public boolean contains(GuideProbe guider) {
-        return guideTargets.exists(GuideProbeTargets.match(guider));
+        return guideTargets.exists(gpt -> gpt.getGuider() == guider);
```

Comment:
Yes, that's right. The static `match` method, I believe, on `GuideProbeTargets` was doing exactly that, but seemed redundant with lambda expressions.